config/locales/en.yml

```diff
@@ -486,3 +486,11 @@ en:
     transparency: See all our transparency data
     written_statement: See all our written statements to Parliament
     authored_article: See all our authored articles
+  broken_links:
+    broken: 
+      title: Broken links
+      subheading: There are broken links in this document
+    caution:
+      title: Links with warnings
+      subheading: There are links with warnings in this document
```

Comment:
This could be "We've found some issues with links that may indicate problems, you should apply caution in linking to them:"